lib/resque/server/views/statuses.erb

```diff
@@ -12,31 +16,43 @@
   <form method="POST" action="<%= u(:statuses) %>/clear/failed" class='clear-failed'>
     <input type='submit' name='' value='Clear Failed Statuses' onclick='return confirm("Are you absolutely sure? This cannot be undone.");' />
   </form>
+  <div class="filters">
+  <h4>Filters</h4>
+  <span class="filter">Status</span>
+    <select id="status_filter">
+    <option></option>
+    <% status_filters.each do |status_filter| %>
+      <option <%= status_filter == filters[:status] ? 'selected' : nil %> value=<%= status_filter %>><%= status_filter.capitalize %></option>
+    <% end %>
+    </select>
+    <span class="filter">Job Name</span>
+    <input type="text" id="job_name" value="<%= filters[:job] %>" />
+  </div>
 <%end%>
 
 <p class='intro'>These are recent jobs created with the Resque::Plugins::Status class</p>
 <table class="vertically-top">
   <tr>
-    <th>ID</th>
+    <th>Started At</th>
     <th>Name</th>
-    <th>Status</th>
-    <th>Last Updated</th>
-    <th class="progress">% Complete</th>
+    <th class="duration">Duration</th>
+    <th class="status">Status</th>
```

Comment:
So `Last Updated` was actually always `start time`?! That's stupid bug.

Reply:
yup, looks like it